File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/City.java
package com.riders.thelab.data.remote.dto.weather;

import com.google.gson.annotations.SerializedName;
import com.squareup.moshi.Json;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@ToString
public class City {

    private int id;
    private String name;
    private String state;
    private String country;

    @Json(name = "coord")
    private Coordinates coordinates;
}

File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/City.kt
package com.riders.thelab.data.remote.dto.weather

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class City(
    @Json(name = "id")
    val id: Int,
    @Json(name = "name")
    val name: String,
    @Json(name = "state")
    val state: String,
    @Json(name = "country")
    val country: String,
    @Json(name = "coord")
    val coordinates: Coordinates
)